kolibri/core/assets/src/core-getters.js

```diff
@@ -0,0 +1,14 @@
+
+const UserKinds = require('./constants').UserKinds;
+
+function isAdminOrSuperuser(state) {
+  const kind = state.core.session.kind;
+  if (kind[0] === UserKinds.SUPERUSER || kind[0] === UserKinds.ADMIN) {
```

Comment:
I'm going to leave this one as-is. [readability > conciseness](https://arpytoth.com/2015/11/15/programming-tips-dont-be-clever/) :)